rmgpy/solver/base.pyx

```diff
@@ -497,10 +497,11 @@ cdef class ReactionSystem(DASx):
                 try:
```

Comment:
not introduced by this PR, but do you mind renaming the variable `l` to comply with PEP 8?
https://www.python.org/dev/peps/pep-0008/#names-to-avoid

Reply:
fixed